docs/65_dandi_hub.md

```diff
@@ -0,0 +1,6 @@
+The DANDI ecosystem includes a self-hosted Jupyter notebook service. This service is orchestrated on a Kubernetes (k8s) cluster
```

Comment:
@asmacdo would be great if you could review

Reply:
With @kabilar 's suggestions, LGTM